Earthfile

```
@@ -99,7 +99,7 @@ integration-test-mysql:
         --pull "mysql:$MYSQL"
         RUN set -e; \
             timeout=$(expr $(date +%s) + 30); \
-            docker run --name mysql --network=host -d -e MYSQL_ROOT_PASSWORD=root "mysql:$MYSQL"; \
+            docker run --name mysql --network=host -d -e MYSQL_ROOT_PASSWORD=root "mysql:$MYSQL" --sql_mode="ANSI_QUOTES"; \
```

```suggestion
            docker run --name mysql --network=host -d -e MYSQL_ROOT_PASSWORD=root "mysql:$MYSQL" --sql_mode="ANSI_QUOTES,STRICT_TRANS_TABLES'"; \
```

This is why the tests don't raise errors anymore. Without STRICT_TRANS_TABLES it lets you insert null values into fields with defaults.

---

So you say `STRICT_TRANS_TABLES` is (part of) the default SQL_MODE, and my command override that? I'll check what the default was, and make sure I _add_ `ANSI_QUOTES` instead, then 👍

---

I've just added the default modes _plus_ the `ANSI_QUOTES`.
Thanks for pointing this out, @greg-rychlewski!